pkg/policy/defaultRules.yaml

```diff
@@ -881,3 +881,314 @@ rules:
                       anyOf:
                         - pattern: ^(169\.254\.)
                         - pattern: ^(127\.)
+  - id: 34
```

Comment:
should start from 35 (last existing default rule is 34)

Reply:
its an old rule, mine start from 35

Reply:
see line 910 pls

Reply:
yeah I know. you accidently added the same rule twice

Reply:
when you're right, you're not wrong